bundles/org.openhab.core/src/main/java/org/openhab/core/util/DurationUtils.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.util;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Static utility methods that are helpful when dealing with {@link Duration}.
+ *
+ * @author Jimmy Tanagra - Initial contribution
+ */
+@NonNullByDefault
+public class DurationUtils {
+
+    private static final Pattern DURATION_PATTERN = Pattern.compile("""
+            (?:([0-9]+)\\s*(?:d|days?))?
```

Comment:
If you google "alternate unit symbol for day" (resp. second, hour, minute, etc.) it suggests "day", "sec", "hour", "min" -- without terminating "s"..

Reply:
.. perhaps the `s?` in the regex does allow them without the terminating s ??